src/dapps/ens-manager-dapp/TheENSManagerLayout.vue

```diff
@@ -504,7 +504,6 @@ export default {
       this.loadingCommit = false;
       this.name = '';
       this.nameHandler = {};
-      this.$refs.registerDomain.reset();
```

Comment:
but dont we need to reset the field?

Reply:
I don't think so since as soon as you click close, the instantiation gets destroyed.  When you reopen, it just uses the default data in ModuleRegisterDomain.vue which starts them off at step 1

`data() {
    return {
      duration: 1,
      onStep: 1,`